traffic_portal/test/integration/PageObjects/LoginPage.po.ts

```diff
@@ -29,24 +29,38 @@ export class LoginPage extends BasePage{
     private config = require('../config');
     private randomize = this.config.randomize;
 
-    async Login(userName: string, password: string ){
-        if(userName == 'admin'){
-            await this.txtUserName.sendKeys(userName)
-            await this.txtPassword.sendKeys(password)
+    async Login(login){
+        let result = false;
+        let basePage = new BasePage();
```

Comment:
`basePage` is never modified, could be `const`

Reply:
Changed basePage to `const`